.github/workflows/stale.yaml

```
@@ -0,0 +1,18 @@
+name: 'Cierra issues y PRs antiguos'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-pr-label: 'needs decision'
+          stale-issue-label: 'stale'
+          days-before-issue-stale: 10
+          days-before-pr-stale: 7
+          days-before-pr-close: 21
+          stale-issue-message: 'Hola, este issue lleva un tiempo sin novedades. Estás trabajando todavía?\nSi necesitas ayuda :sos: no dudes en contactarnos en nuestro grupo de Telegram.'
```

```suggestion
          stale-issue-message: 'Este issue lleva un tiempo sin actualizaciones. ¿Estás trabajando todavía?\nSi necesitas ayuda :sos: no dudes en contactarnos en nuestro [grupo de Telegram](https://t.me/python_docs_es).'
```
No sé si funcionará el enlace como markdown.

---

Vamos a probarlo y ya hacemos cambios si no funciona.